fava/templates/overlays/_transaction_form.html

```diff
@@ -12,20 +12,21 @@
                 <label for="payee">Narration:</label>
                 <input type="text" name="narration" placeholder="Narration" />
             </div>
-            <div class="field-control posting">
-                <label for="account">Account:</label>
-                <input type="text" class="account" name="account" placeholder="Account" list="active-accounts" />
-                <input type="number" step="0.01" class="number" name="number" placeholder="Number" />
-                <input type="text" class="currency" name="currency" placeholder="Currency" list="currencies" />
+            <div class="postings">
+            {% for i in range(15) %}
```

Comment:
This seems like a waste of bandwidth. [cloneNode](https://developer.mozilla.org/en/docs/Web/API/Node/cloneNode) should do the trick